src/browserbehavior/DefaultBrowserBehavior.ts

```diff
@@ -90,15 +109,16 @@ export default class DefaultBrowserBehavior implements BrowserBehavior {
   }
 
   requiresCheckForSdpConnectionAttributes(): boolean {
-    return !this.isIOSSafari();
+    return !this.isIOSSafari() && !this.isIOSChrome() && !this.isIOSFirefox();
   }
 
   requiresIceCandidateGatheringTimeoutWorkaround(): boolean {
     return this.hasChromiumWebRTC();
   }
 
   requiresUnifiedPlanMunging(): boolean {
-    let shouldRequire = this.isSafari() && this.isUnifiedPlanSupported();
+    let shouldRequire =
+      (this.isSafari() && this.isUnifiedPlanSupported()) || this.hasWebKitWebRTC();
```

Comment:
Doesn't Safari also use WebKit? Should we group this under something like WebKitBasedBrowsers (similar to ChromiumBasedBrowsers?

Reply:
Yes, macOS and iOS Safari are WebKit-based browsers

Reply:
Added Safari to the array of Webkit browsers